projects/aca-content/src/lib/components/header/header.component.spec.ts

```diff
@@ -85,38 +82,11 @@ describe('AppHeaderComponent', () => {
   it('should set header color, header text color, name and logo', fakeAsync(() => {
     component.appName$.subscribe((val) => expect(val).toBe(app.appName));
     component.logo$.subscribe((val) => expect(val).toBe(app.logoPath));
-    component.headerColor$.subscribe((val) => expect(val).toBe(app.headerColor));
-    component.headerTextColor$.subscribe((val) => expect(val).toBe(app.headerTextColor));
   }));
 
   it('should get header actions', fakeAsync(() => {
     component.ngOnInit();
     tick();
     expect(component.actions).toEqual(actions);
   }));
-
-  describe('Search input', () => {
-    beforeEach(() => {
-      localStorage.clear();
-    });
-
-    afterEach(() => {
-      localStorage.clear();
-    });
-
-    it('should search be present when contentService is enabled', () => {
```

Comment:
Why did you removed this unit tests? Nothing changed with aca search within this component as far as I can see

Reply:
the whole component is removed as no longer used